src/main/java/org/mitre/inferno/rest/ValidatorEndpoint.java

```diff
@@ -39,11 +39,30 @@ public static ValidatorEndpoint getInstance(Validator validator) {
    * Creates the API routes for receiving and processing requests for the validation service.
    */
   private void createRoutes() {
+
+    post("/*/%24validate",
+    (req, res) -> {
+      res.type("application/fhir+json");
+      return validateResource(req.bodyAsBytes(), req.queryParams("profile"));
+    });
+
+    post("/*/*/%24validate",
+    (req, res) -> {
+      res.type("application/fhir+json");
+      return validateResource(req.bodyAsBytes(), req.queryParams("profile"));
+    });
+
+    post("/*/validate",
+    (req, res) -> {
+      res.type("application/fhir+json");
+      return validateResource(req.bodyAsBytes(), req.queryParams("profile"));
+    });
```

Comment:
```suggestion
```


I don't believe this endpoint is necessary, as there is no need to check for wildcards on calls to `validate`. Any call to `validate` will be captured by the route on line 61. Think of any wildcard path preceding `validate` as incorrect. For example, the wrapper does not need to handle a request made to `[base]/[resource]/[id]/validate` since we've decided that the API is only responsible for `[base]/validate` -- the inclusion of `[resource]/[id]` simply makes it an unsupported path.